src/components/ProfileForm/components/ProfileFormStep1.tsx

```diff
@@ -0,0 +1,148 @@
+const ProfileFormStep1 = () => {
+  return (
+    <>
+      <form className="flex justify-around">
```

Comment:
Same comment as above. Remove the `form` this should be in the parent component.

NOTE: same applies for the rest.